packages/spec/integration/graphql-no-ssr/__tests__/develop.js

```diff
@@ -0,0 +1,16 @@
+describe('graphql mock server without SSR', () => {
+  let url;
+
+  beforeAll(async () => {
+    url = await HopsCLI.start('--fast-dev');
+  });
+
+  it('renders the page correctly', async () => {
+    const { page, getInnerText } = await createPage();
+    await page.goto(url, { waitUntil: 'networkidle2' });
+
+    expect(await getInnerText('p')).toBe('Hello World!');
```

Comment:
Shouldn't we also assert that:
* nothing was rendered server-side
* no fetch requests have been dispatched on the server?

Reply:
Yeah, it should check for `Loading...`. Will add that.

The query-part is actually done, but a little disguised. Only the first query will receive the `subject` `"World"`. All subsequent queries receive `"SSR"`. So the test will fail, when there has been a server-side query prior to the one on the client.

Reply:
Okay, switched the wording to `Client` and `Server`. So when the text of the rendered app says `Hello Server!`, something went wrong, a request has been made on the server & the test will notify us.